b64json/b64json.go

```diff
@@ -3,8 +3,37 @@ package b64json
 import (
 	"encoding/base64"
 	"encoding/json"
+	"github.com/iancoleman/orderedmap"
 )
 
+func indexOf(needle string, data []string) int {
+	for k, v := range data {
+		if needle == v {
+			return k
+		}
+	}
+	return -1
+}
+
+func rubyJsonSort(a *orderedmap.Pair, b *orderedmap.Pair) bool {
```

Comment:
Just call it sort, It has nothing to do with ruby, it's to be considered arbitrary at this point

Reply:
Sure thing.